### PR TITLE
Add notes for Surah Naaziyat and Surah Abasa

### PR DESCRIPTION
**Pull request by mushtaqak**
This PR adds notes for Surah Naaziyat and Surah Abasa.

### CODE REVIEW

**Comment by mushtaqak** [comment]
@miankhalid This is also ready for reviewing :)

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ This repo contains the notes devised by me after listening to Dr. Israr Sahab's
 - [16. Surah Nahal](notes/roman-urdu/16_Surah_Nahal.md)
 - [17. Surah Bani Israel](notes/roman-urdu/17_Surah_Bani_Israel.md)
 - [18. Surah Kahf](notes/roman-urdu/18_Surah_Kahf.md)
+- [79. Surah Nazi'at](notes/roman-urdu/79_Surah_Nazi'at.md)
+- [80. Surah Abas](notes/roman-urdu/80_Surah_Abas.md)
```
Comment (miankhalid) [inline README.md]: Its `Surah Abasa`

File: notes/roman-urdu/79_Surah_Nazi'at.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Number 79: Surah Nazi'at
+
+Classification: Makki  
+Rukus: 2  
+Ayahs: 46  
+Subject: Resurrection  
+Related Surah: Surah Abas  
+
+‌Farishto ki qasma, rooh qabz karne waly.
```
Comment (miankhalid) [inline notes/roman-urdu/79_Surah_Nazi'at.md]: qasam instead of qasma.

File: notes/roman-urdu/79_Surah_Nazi'at.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Number 79: Surah Nazi'at
+
+Classification: Makki  
+Rukus: 2  
+Ayahs: 46  
+Subject: Resurrection  
+Related Surah: Surah Abas  
+
```
Comment (miankhalid) [inline notes/roman-urdu/79_Surah_Nazi'at.md:8]: Is there a background to this surah?

**Review by miankhalid — CHANGES_REQUESTED** [review]
Same comments as last notes i.e.

1. Better to divide the whole notes doc into multiple headings based on topics and
2. have ordered list of points under them

File: notes/roman-urdu/79_Surah_Nazi'at.md (hunk 3 above)
Comment (mushtaqak) [reply notes/roman-urdu/79_Surah_Nazi'at.md:8]: I have added background based on Dr. Israr's video. We can add background to any Surah if we know add study them better. For now, I would say no. Does this makes sense?

**Comment by mushtaqak** [comment]
@miankhalid I have addressed your concerns, kindly have a look.

**Review by miankhalid — APPROVED** [review]

**Pull request merged**